1-JavaScript/Week3/homework/ex3-tellFortune.js

```diff
@@ -25,30 +25,20 @@ Note: The DRY is put into practice here: instead of repeating the code to
 randomly select array elements four times inside the `tellFortune` function 
 body, this code is now written once only in a separated function.
 -----------------------------------------------------------------------------*/
-const numKids = [
-  // TODO add elements here
-];
-
-const partnerNames = [
-  // TODO add elements here
-];
-
-const locations = [
-  // TODO add elements here
-];
-
-const jobTitles = [
-  // TODO add elements here
-];
-
+const numKids = [1,2,3,4,5];
+const partnerNames = ['Hannan','Aya','Asel','Mila','Marim'];
+const locations = ['Amsterdam','Cairo','Pairs','London','NewYork'];
+const jobTitles = ['train driver','rock star','spy','serial killer','movie star'];
 // This function should take an array as its parameter and return
 // a randomly selected element as its return value.
-function selectRandomly(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function selectRandomly(arr) {
+const randomElement = arr[Math.floor(Math.random() * arr.length)];
+return randomElement;
 }
-
-function tellFortune(/* add parameter(s) here */) {
-  // TODO complete this function
+function tellFortune(numKids, partnerNames, locations, jobTitles) {
+  
+  const tellFortuneMessage = `You will be a ${ selectRandomly(jobTitles)} in ${selectRandomly(locations)}, married to ${selectRandomly(partnerNames)} with ${selectRandomly(numKids)} kids.`
+  return tellFortuneMessage
```

Comment:
remember to add semicolons at the end of each statement.

```suggestion
  const tellFortuneMessage = `You will be a ${ selectRandomly(jobTitles)} in ${selectRandomly(locations)}, married to ${selectRandomly(partnerNames)} with ${selectRandomly(numKids)} kids.`;
  
  return tellFortuneMessage;
```